src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -40,14 +40,19 @@
   <Choose>
     <When Condition="$(TargetFramework) == 'netstandard2.0'">
       <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+        <!-- We have to ship pre-patch versions of NewtonSoft for VisualStudio SDK. 
+             We mitigate risk by limiting nesting depth. -->
+        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" NoWarn="NU1903" />
+
         <PackageReference Include="System.IO.FileSystem.Primitives" Version="4.3.0" />
         <PackageReference Include="System.Text.Encoding.Extensions" Version="4.3.0" />
       </ItemGroup>
     </When>
     <Otherwise>
       <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="8.0.1" />
+        <!-- We have to ship pre-patch versions of NewtonSoft for VisualStudio SDK. 
```

Comment:
>patch [](http://example.com/codeflow?start=33&length=5)

This reference is for .net framework builds. For this condition, I think our intent was to take the maximal version for .NET framework only. But does that even make sense? Also, I see that even for this standard, 8.0.3 is the last version.

So minimally, we should update this reference to 8.0.3. I honestly couldn't say without more research whether we actually need to drop to 8.X for this case.